libs/blocks/modal/modal.js

```diff
@@ -110,6 +110,29 @@ export async function sendViewportDimensionsOnRequest(messageInfo) {
   }
 }
 
+/** For the modal height adjustment to work the following conditions must be met:
+ * 1. The modal must have classes 'commerce-frame height-fit-content';
+ * 2. The iframe inside must send a postMessage with the contentHeight (a number of px or '100%);
+ */
+function adjustModalHeight(contentHeight) {
+  if (!contentHeight || !window.location.hash) return;
+  const dialog = document.querySelector(window.location.hash);
```

Comment:
Passing `window.location.hash` to the `querySelector` method as is, seems to be  very fragile to me.
For instance, Milo Commerce code uses `window.location.hash` to store some state in format of a query string (e.g. `#a=1&b=2`).

Reply:
Thanks, changed it to not operate the window.location.hash